docs/QSG/qemu-vms.md

```diff
@@ -0,0 +1,284 @@
+```
+    sudo qemu-system-x86_64 -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 3 -m 12288 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-server.qcow2,if=virtio -device virtio-net,netdev=mynet0,mac=52:54:00:12:34:56 -netdev tap,id=mynet0 -drive file=<image-dir>/qemu-nvm-disk1.qcow2,if=none,id=nvm1 -device nvme,serial=deadbeef,drive=nvm1 -cdrom <iso-dir>/Rocky-8.9-x86_64-minimal.iso -boot menu=on &
+    sudo qemu-system-x86_64  -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 1 -m 2048 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-client.qcow2,if=virtio -device virtio-net,netdev=mynet1,mac=52:54:00:12:34:57 -netdev tap,id=mynet1 -cdrom <iso-dir>/Rocky-8.9-x86_64-minimal.iso -boot menu=on
+```
+4.  Install DAOS software.
+
+After OS installation is finished. Remove the `-cdrom` and `-boot` options from the QEMU commands to boot VMs normally. Next is to install DAOS software.
+```
+    sudo qemu-system-x86_64 -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 3 -m 12288 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-server.qcow2,if=virtio -device virtio-net,netdev=mynet0,mac=52:54:00:12:34:56 -netdev tap,id=mynet0 -drive file=<image-dir>/qemu-nvm-disk1.qcow2,if=none,id=nvm1 -device nvme,serial=deadbeef,drive=nvm1 &
+    sudo qemu-system-x86_64  -M q35,accel=kvm,kernel-irqchip=split -cpu host -smp 1 -m 2048 -device intel-iommu,intremap=on -drive file=<image-dir>/daos-client.qcow2,if=virtio -device virtio-net,netdev=mynet1,mac=52:54:00:12:34:57 -netdev tap,id=mynet1 
+```
+I follow these [steps](https://docs.daos.io/v2.4/QSG/setup_rhel/#start-the-daos-agents) to install both the DAOS server, DAOS admin, and DAOS client. I install daos-server on the first VM, and install both daos-admin and daos-client on the second VM. If you have issues with Ceph Pacific Repo you can edit `/etc/yum.repos.d/CentOS-Ceph-Pacific.repo` by commenting out mirrorlist and changing baseurl as below.
+```
+...
+[centos-ceph-pacific]
+name=CentOS-$stream - Ceph Pacific
+#mirrorlist=http://mirrorlist.centos.org/?release=$stream&arch=$basearch&repo=storage-ceph-pacific
+baseurl=http://buildlogs.centos.org/centos/$stream/storage/$basearch/ceph-pacific/
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Storage
+...
+```
+After DAOS installation you still need to update the config files.
+
+5.  Update config files.
+
+Update the daos-server config file `/etc/daos/daos_server.yml` on daos-server. You may need to change fabric\_iface and bdev\_list if they are different.
+```
+    name: daos_server
+    access_points:
+    - daos-server
+    port: 10001
+    
+    nr_hugepages: 1024
+    system_ram_reserved: 4
+    disable_vfio: true 
+    
+    transport_config:
+        allow_insecure: false
+        client_cert_dir: /etc/daos/certs/clients
+        ca_cert: /etc/daos/certs/daosCA.crt
+        cert: /etc/daos/certs/server.crt
+        key: /etc/daos/certs/server.key
+    provider: ofi+sockets
```

Comment:
ofi+tcp is preferred over sockets

Reply:
From an undefned reasons ofi+tcp and ofi+tcp;ofi_rxm were not working.

Reply:
I experimented with ofi+tcp and ofi+tcp;ofi_rxm. These two options are not working. Since this setup is used for experimenting with DAOS, I have to use ofi+sockets for now.

Reply:
I add a comment to explain current dilemma.